src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml

```diff
@@ -56,6 +56,7 @@ sylius:
         create_a_new_customer_account: Create a new customer account
         if_you_have_an_account_sign_in_with_your_email_address: If you have an account, sign in with your e-mail address
         access_your_store_account_or_create_a_new_one: Access your store account or create a new one
+        canceled_payment: Your payment was not successful.
```

Comment:
Translation key should be exactly the same as message (with dashes instead of whitespaces). Same for below keys.